src/controller/routeController.js

```diff
@@ -0,0 +1,69 @@
+import model from '../database/models';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Do not disable eslint rules. You should rather conform to them

Reply:
Now it is corrected